src/posts/entities/option.repository.ts

```diff
@@ -14,10 +14,12 @@ export class OptionRepository extends Repository<Option> {
     group: OptiosnGroup,
   ): Promise<Option[]> {
     // create option object with each optionData
-    const options = optionsData.map((option) => {
+    const options = optionsData.map((option, index) => {
       const newOption = this.create();
       newOption.body = option.body;
       newOption.vote_count = 0;
+      newOption.order = index + 1;
```

Comment:
why not zero-based order as you already set the default order in the entity to be 0?

Reply:
Numbering the postgres DB start any table from 1, so we thought it would be better to follow them, i.e. use 1 indexed.
Beside that, we used 0 as a default value for previous posts that has been created but don't have any order set for them.

Reply:
I didn't get the postgres part. does postgres rejects 0 as number?

Reply:
By default postgres starts from 1, so only thought it would be better to do the same.

Reply:
This's an example for what I mean:
![Screenshot from 2021-06-24 19-50-44](https://user-images.githubusercontent.com/12779125/123309762-9d083880-d525-11eb-9faa-0698c1348bfc.png)

It starts counting/numbering from 1, so just thought of following them.
If this is not good, so surely I will start from 0, but which order number to give for previous posts that has been already created?

Reply:
could you please give it another thought and think about what "start with"? this is just a column where we set whatever number we want and later we use to sort the options. this is not a sequence or incremental number.

I truly didn't get your point.

Reply:
I understood. Done.